cohttp-lwt-unix/src/net.mli

```diff
@@ -16,14 +16,13 @@
 
 (** Basic satisfaction of {!Cohttp_lwt.Net} *)
 
-module IO = Io
-
 type ctx = { ctx : Conduit_lwt_unix.ctx; resolver : Resolver_lwt.t }
 [@@deriving sexp_of]
 
-val default_ctx : ctx
-(** [default_ctx] is the default network context. It uses
-    [Conduit_lwt_unix.default_ctx] and [Resolver_lwt_unix.system]. *)
+include Cohttp_lwt.S.Net
+  with module IO = Io
+   and type ctx := ctx
+   and type endp = Conduit.endp
```

Comment:
Why don't we use a destructive substitution for `endp` as well?

Reply:
Because the interface missing a `endp` type would no longer satisfy `Cohttp_lwt.S.Net`. I can and need to do the destructive substitution only because left and right-hand-side bear the same name.